lib/storage/client.js

```diff
@@ -1143,6 +1205,46 @@ export default class Client extends CryptoConsumer {
     return response.json()
   }
 
+  /**
+   * Allow another client to grant read access this client's records of a specific type
+   *
+   * @param {string} authorizerId The client ID to grant sharing privileges to
+   * @param {string} type The record type to grant sharing access for
+   *
+   * @returns {Promise<bool>} Whether the authorizer was added successfully.
+   */
+  async addAuthorizer(authorizerId, type) {
+    const clientId = this.config.clientId
+    if (authorizerId === clientId) {
+      return true
+    }
+
+    await putAccessKeyForReader(this, clientId, type, authorizerId)
+
+    const policy = { allow: [{ authorizer: {} }] }
+    return putPolicy(this, policy, clientId, clientId, authorizerId, type)
+  }
+
+  /**
+   * Remove another client's ability to grant read access this client's records of a specific type
+   *
+   * @param {string} authorizerId The client ID to revoke sharing privileges from
+   * @param {string} type The record type to revoke sharing access for
+   *
+   * @returns {Promise<bool>} Whether the authorizer was removed successfully.
+   */
+  async removeAuthorizer(authorizerId, type) {
+    const clientId = this.config.clientId
+    if (authorizerId === clientId) {
+      return true
```

Comment:
should this be `false`? Can an authorizer remove themselves?

Reply:
I believe AddAuthorizer has the same "return true" no-op; Maybe both should return false to indicate this operation is doing nothing?